language/move-prover/src/bytecode_translator.rs

```diff
@@ -966,20 +945,19 @@ impl<'env> ModuleTranslator<'env> {
             SpecBlock(_, block_id) => {
                 self.generate_function_spec_inside_impl(func_target, *block_id);
             }
-            Label(_, label) => {
+            Bytecode::Label(_, label) => {
                 self.writer.unindent();
                 emitln!(self.writer, "L{}:", label.as_usize());
                 if ctx.loop_targets.contains_key(label) {
                     emitln!(self.writer, "assume !$abort_flag;");
-                    emitln!(self.writer, "assume $ExistsTxnSenderAccount($m, $txn);");
                     let targets = &ctx.loop_targets[label];
                     for idx in 0..func_target.get_local_count() {
                         if let Some(ref_proxy_idx) = func_target.get_ref_proxy_index(idx) {
                             if targets.contains(ref_proxy_idx) {
                                 let ref_proxy_var_name = str_local(*ref_proxy_idx);
                                 let proxy_idx = func_target.get_proxy_index(idx).unwrap();
                                 emitln!(self.writer,
-                            "assume l#$Reference({}) == $Local({}) && p#$Reference({}) == $EmptyPath;",
+                            "assume l#$Mutation({}) == $Local({}) && p#$Mutation({}) == $EmptyPath;",
```

Comment:
I was wondering about the reason for this renaming Reference -> Mutation.  Perhaps you wanted to indicate that this  is reference is really a mutable reference.  Did you consider MutableReference?

Reply:
If find both Reference and MutRef confusing, because the thing carries the value which is mutated. Under this view, Mutation made more sense to me (after playing with other options a while)